/**:
  ros__parameters:
    encoder_onnx_path: "$(var model_path)/pts_voxel_encoder_$(var model_name).onnx"
    encoder_engine_path: "$(var model_path)/pts_voxel_encoder_$(var model_name).engine"
    head_onnx_path: "$(var model_path)/pts_backbone_neck_head_$(var model_name).onnx"
    head_engine_path: "$(var model_path)/pts_backbone_neck_head_$(var model_name).engine"
    trt_precision: fp16
    cloud_capacity: 2000000
    post_process_params:
      # post-process params
      circle_nms_dist_threshold: 0.3
      iou_nms_search_distance_2d: 10.0
      iou_nms_threshold: 0.1
      score_threshold: 0.35
      yaw_norm_thresholds: [0.3, 0.3, 0.3, 0.3, 0.0]
    densification_params:
      world_frame_id: "map"
      num_past_frames: 1
    omp_params:
      # omp params
      num_threads: 1
